core/src/main/java/org/infinispan/configuration/global/GlobalTracingConfiguration.java

```diff
@@ -14,9 +14,10 @@ public class GlobalTracingConfiguration {
    public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.ENABLED, true, Boolean.class).immutable().build();
```

Comment:
We may want to evaluate whether this is enabled or not... My guess is that majority of users will never use global tracing and it probably should be disabled by default as it will have an impact to performance.